packages/@storefront/sort/src/sort/index.ts

```diff
@@ -34,21 +34,35 @@ class Sort {
     this.updateSorts();
   }
 
+  onUpdate() {
+    this.state = {
+      ...this.state,
+      sorts: this.extractSorts(),
+    };
+  }
+
   updateSorts = () => this.set({ sorts: this.extractSorts() });
 
   extractSorts() {
     let sorts;
+    let labels;
```

Comment:
Is it more legible to have this default to `sorts.labels`, then conditionally set `labels` inside the switch?

Reply:
Will investigate.

Reply:
This feedback item may have been addressed by the introduction of `extractLabels()`. Please re-review.